docs/content.zh/docs/connectors/table/jdbc.md

```diff
@@ -663,20 +668,22 @@ Flink 支持连接到多个使用方言（dialect）的数据库，如 MySQL、O
         <code>BIGINT</code><br>
         <code>BIGSERIAL</code></td>
       <td><code>BIGINT</code></td>
+      <td>
+        <code>INTEGER</code><br>
+        <code>INT</code><br>
+        <code>BIGINT</code><br>
+        <code>INT8</code><br>
+        <code>SMALLINT</code><br>
+        <code>TINYINT</code></td>
       <td><code>BIGINT</code></td>
     </tr>
    <tr>
       <td><code>BIGINT UNSIGNED</code></td>
       <td></td>
       <td></td>
       <td></td>
-      <td><code>DECIMAL(20, 0)</code></td>
```

Comment:
Why we are removing this?

Reply:
I did a little cleanup here, no information was removed(though it looks like that in the change log).
This is what it currently looks like on the **master** branch for the Chinese version(3 table rows instead of 2):
```
<tr>
  <td>
    <code>BIGINT</code><br>
    <code>INT UNSIGNED</code></td>
  <td></td>
  <td>
    <code>BIGINT</code><br>
    <code>BIGSERIAL</code></td>
  <td><code>BIGINT</code></td>
  <td><code>BIGINT</code></td>
</tr>
<tr>
  <td><code>BIGINT UNSIGNED</code></td>
  <td></td>
  <td></td>
  <td></td>
  <td><code>DECIMAL(20, 0)</code></td>
</tr>
<tr>
  <td><code>BIGINT</code></td>
  <td></td>
  <td><code>BIGINT</code></td>
  <td><code>BIGINT</code></td>
</tr>
```
My change merged two BIGINT rows and it looks like this(2 table rows, as in the English version):
```
<tr>
  <td>
    <code>BIGINT</code><br>
    <code>INT UNSIGNED</code></td>
  <td></td>
  <td>
    <code>BIGINT</code><br>
    <code>BIGSERIAL</code></td>
  <td><code>BIGINT</code></td>
  <td>
    <code>INTEGER</code><br>
    <code>INT</code><br>
    <code>BIGINT</code><br>
    <code>INT8</code><br>
    <code>SMALLINT</code><br>
    <code>TINYINT</code></td>
  <td><code>BIGINT</code></td>
</tr>
<tr>
  <td><code>BIGINT UNSIGNED</code></td>
  <td></td>
  <td></td>
  <td></td>
  <td></td>
  <td><code>DECIMAL(20, 0)</code></td>
</tr>
```